presto-iceberg/src/main/java/com/facebook/presto/iceberg/IcebergAbstractMetadata.java

```diff
@@ -434,4 +434,36 @@ public Optional<SystemTable> getSystemTable(ConnectorSession session, SchemaTabl
 
         return getIcebergSystemTable(tableName, icebergTable);
     }
+
+    /**
+     * Deletes the files from a particular scan
+     *
+     * @return
+     */
+    private long deleteTable(Table icebergTable, Iterable<FileScanTask> scan)
```

Comment:
With the new `truncateTable` function, the name of this function (`deleteTable`) looks a bit misleading. How about `deleteFiles`? Or any other ideas?

Reply:
I've updated the name to `removeScanFiles`